ai-alert-backend/.env.example

```
@@ -0,0 +1,25 @@
+
+# Server Configuration
+PORT=3001
+NODE_ENV=development
+
+# External API Keys (Optional for MVP)
+COINGECKO_API_KEY=
+ALPHAVANTAGE_API_KEY=
+
+# Security
+JWT_SECRET=your_jwt_secret_here
+CORS_ORIGINS=http://localhost:3000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CORS_ORIGINS key should go before the JWT_SECRET key

(UnorderedKey)